modules/signalfx/src/main/java/com/spotify/ffwd/signalfx/SignalFxOutputPlugin.java

```diff
@@ -103,7 +105,7 @@ public Module module(final Key<PluginSink> key, final String id) {
             protected void configure() {
                 final Key<SignalFxPluginSink> sinkKey =
                     Key.get(SignalFxPluginSink.class, Names.named("signalfxSink"));
-                bind(sinkKey).to(SignalFxPluginSink.class);
+                bind(sinkKey).to(SignalFxPluginSink.class).in(Scopes.SINGLETON);
```

Comment:
this limits possibility to declare multiple signalfx output plugins

Reply:
This was a deliberate decision on the input of @udoprog. I believe that the reasoning is that there's no need to have multiple such output plugins specified.